OpenSim/Simulation/OpenSense/InverseKinematicsStudy.cpp

```diff
@@ -113,18 +113,21 @@ runInverseKinematicsWithOrientationsFromFile(Model& model,
     TableReporter* ikReporter = new TableReporter();
     ikReporter->setName("ik_reporter");
     auto coordinates = model.updComponentList<Coordinate>();
-    std::vector<std::string> transCoordNames;
+    std::vector<std::string> rotCoordNames;
 
     // Hookup reporter inputs to the individual coordinate outputs
-    // and keep track of coordinates that are translations
+    // and keep track of coordinates that are rotational
     for (auto& coord : coordinates) {
         ikReporter->updInput("inputs").connect(
             coord.getOutput("value"), coord.getName());
-        if (coord.getMotionType() == Coordinate::Translational) {
-            transCoordNames.push_back(coord.getName());
-            coord.set_locked(true);
+        if (coord.getMotionType() == Coordinate::Rotational) {
+            rotCoordNames.push_back(coord.getName());
+        }
+        else if(coord.getMotionType() == Coordinate::Translational) {
+            coord.setDefaultLocked(true);
         }
```

Comment:
Is there difference in behavior between set_locked and setDefaultLocked?

Reply:
No difference. `setDefaultLocked` is more explicit about the default state of the coordinate is locked.